### PR TITLE
Implement a local configuration of the database

### PR DESCRIPTION
**Pull request by mecoccaro**
Issue #323 

Added to the package.json file an extra script for Windows users, so they can run the project without having to modify the normal `npm run server`. Modified the README file with the new instructions for Windows users and added instructions for use of a local database.

Implementation proof

https://user-images.githubusercontent.com/36824674/112883704-a146c600-909c-11eb-8c7f-08cad70e7262.mp4

Local DB
imagen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,14 +35,36 @@ npm run ui
...
+npm run server-windows
+```
+
 5. UI should open in the browser at `localhost:6002`
 
 If using VSCode, be sure to disable the Prettier: Format on Save feature to prevent unwanted changes to the code styles.
 
+## Using a local database 
+
+If you want to set a local databe follow this steps:
```
Comment (otech47) [inline README.md:54]: Fix typo: `set up a local database, follow these steps:`

File: README.md [hunk 2]
```diff
@@ -35,14 +35,36 @@ npm run ui
...
+
+1. Download MySQL server (Usually it comes with others features).
```
Comment (otech47) [inline README.md:56]: Maybe provide a link here to help the reader find the correct version of MySQL to download?

File: README.md [hunk 3]
```diff
@@ -35,14 +35,36 @@ npm run ui
...
+
+2. Configure your environment.
+
+3. Create a connection.
+
+4. Create a schema.
```
Comment (otech47) [inline README.md:62]: These 3 steps are a bit too general for someone inexperienced. Does this involve running commands? It's nice and convenient to provide some sample commands for the reader to be able to run on their machine to make sure this step completed properly

File: package.json [hunk 4]
```diff
@@ -57,6 +57,7 @@
   "scripts": {
     "ui": "cross-env PORT=6002 react-scripts start",
     "server": "PORT=6001 node server",
+    "server-windows": "SET PORT=6001 && node server.js",
```
Comment (otech47) [inline package.json:60]: Okay solution for now but I think even better would be a `cross-env` implementation (https://www.npmjs.com/package/cross-env)

File: README.md [hunk 5]
```diff
@@ -35,14 +35,36 @@ npm run ui
...
+
+5. In your `.env` file set the MYSQL variables with your own values.
```
Comment (otech47) [inline README.md:64]: I think this is the 1st time a `.env` file is mentioned in the README. Let's try and add some more detail here and [minimize required knowledge](https://github.com/setlife-network/trinary/blob/develop/CONTRIBUTING.md#1-minimize-required-knowledge)

**Review by otech47 — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (mecoccaro) [reply package.json:60]: I did not know about that implementation, it works perfectly! Thanks.

File: README.md [hunk 6]
```diff
@@ -43,6 +43,47 @@ npm run server
 
 If using VSCode, be sure to disable the Prettier: Format on Save feature to prevent unwanted changes to the code styles.
 
+## Using a local database
```
Comment (otech47) [inline README.md:46]: 🔥

**Review by otech47 — APPROVED** [review]

**Pull request merged**